tests/hipscat/pixel_math/test_margin_bounding.py

```diff
@@ -68,24 +68,31 @@ def test_get_margin_bounds_and_wcs():
     test_ra3 = 50.61225197
     test_dec3 = 14.4767556
 
-    test_ra = np.array([test_ra1, test_ra2, test_ra3])
-    test_dec = np.array([test_dec1, test_dec2, test_dec3])
-
-    test_sc = SkyCoord(test_ra, test_dec, unit="deg")
+    test_sc = SkyCoord(
+        np.array([test_ra1, test_ra2, test_ra3]), np.array([test_dec1, test_dec2, test_dec3]), unit="deg"
```

Comment:
is this caused by the too many local variables rule? for me the first way of doing it is more readable but ¯\_(ツ)_/¯

Reply:
Yes, it was, and I agree. I got a little carried away. 

I've just bumped the overall allowed locals, because I think the original narrative style is more in the spirit of unit testing.